wp/headless-wp/includes/classes/Plugin.php

```diff
@@ -48,6 +48,9 @@ public function register() {
 
 		$gutenberg = new Gutenberg();
 		$gutenberg->register();
+
+		$search = new Search\Search();
+		$search->register();
```

Comment:
It looks like your customization ended up removing the "self" link, so when `_embed` is passed it doesn't embed the full post object anymore.

See; https://js1.10up.com/wp-json/wp/v2/search?_embed here it embeds "self" with the full post object.

Perhaps you just need to add the link for the post/term itself?

Reply:
added back self